### ISSUE TITLE
Is there Hyperparameter Evolution for Classification like Object Detection?

### ISSUE THREAD

**Issue by star4s** [ultralytics/yolov5#10535]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Is there Hyperparameter Evolution for Classification like Object Detection?

I did not fine about augmentation for Classification  like Object Detection.

I just find some options "--optimizer", "--lr0", "--decay", and "--label-smoothing" for Hyperparameter tuning.

**Comment by glenn-jocher** [comment]
@star4s hi, thank you for your feature suggestion on how to improve YOLOv5 🚀! This feature does not currently exist.

The fastest and easiest way to incorporate your ideas into the official codebase is to submit a Pull Request (PR) implementing your idea, and if applicable providing before and after profiling/inference/training results to help us understand the improvement your feature provides. This allows us to directly see the changes in the code and to understand how they affect workflows and performance.

Please see our ✅ [Contributing Guide](https://docs.ultralytics.com/help/contributing/) to get started.
